src/controller/RequestController.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import axios from 'axios';
+import logger from '../logger';
+import type{ convParams } from './DialogflowController';
+
+const url: string = 'https://api.askir.me/images';
+
+export type Image = {
+  id: string,
+  type: string,
+  created_at: string,
+  updated_at: string,
```

Comment:
Personally, I am a fan of converting snake case params from other APIs to javascript camel case.